docs/intro/glossary.md

```diff
@@ -0,0 +1,241 @@
+# Glossary
+
+This glossary provides definitions for common terminology used in the SDK,
+focusing on microgrid components, metrics, measurements, and power-related
+terms.
+
+## Common Acronyms
+
+### AC
+
+Alternating current. See the [Wikipedia article](https://en.wikipedia.org/wiki/Alternating_current) for more details.
+
+### BMS
+
+Battery management system. See the [Wikipedia article](https://en.wikipedia.org/wiki/Battery_management_system) for more details.
+
+### CHP
+
+Combined heat and power. See the [Wikipedia article](https://en.wikipedia.org/wiki/Combined_heat_and_power) for more details.
+
+### DC
+
+Direct current. See the [Wikipedia article](https://en.wikipedia.org/wiki/Direct_current) for more details.
+
+### EV
+
+Electric vehicle. See the [Wikipedia article](https://en.wikipedia.org/wiki/Electric_vehicle) for more details.
+
+### PSC
+
+Passive sign convention. See the [Wikipedia article](https://en.wikipedia.org/wiki/Passive_sign_convention) for more details.
```

Comment:
We should add "Passive sign convention" to the glossary too.

Reply:
True, I just don't know where to put it, as it doesn't fit any section for now. I guess I will create a `Electrical terms` section or something...

Reply:
Done in the Microgrid section, as I didn't want to start a new section if we are not sure about the structure yet.